src/codewhisperer/util/userGroupUtil.ts

```diff
@@ -51,10 +51,7 @@ export class CodeWhispererUserGroupSettings {
     }
 
     private guessUserGroup(): UserGroup {
```

Comment:
help me understand why this enabled crossfile for java users?

Reply:
The userGroup is just the treatment group with feature name on it. So here I remove all usages of crossfile group.